website/content/en/docs/building-operators/golang/advanced-topics.md

```diff
@@ -287,14 +263,20 @@ func main() {
 When the operator is not running in a cluster, the Manager will return an error on starting since it can't detect the operator's namespace in order to create the configmap for leader election. You can override this namespace by setting the Manager's `LeaderElectionNamespace` option.
 
 [typical-status-properties]: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties
+
+<!-- todo(camilamacedo) 
+the following link was comment for do no broke the CI
+this impl probably will came from operator-lib
+however, we also need to check if we would not like to just let them use
+https://github.com/kubernetes/kubernetes/pull/92717/files 
 [godoc-conditions]: https://godoc.org/github.com/operator-framework/operator-sdk/pkg/status#Conditions
```

Comment:
this one is falling now. I am not sure if we will still providing or not the helper in via operator-lib so just comment and raised : https://github.com/operator-framework/operator-sdk/issues/3515